stable/mission-control/templates/jfmc-setup-scripts.yaml

```diff
@@ -45,8 +45,8 @@ data:
         touch  ${JFMC_PROPERTIES} || (  echo "unable to create ${JFMC_PROPERTIES} file" && exit 1 )
     fi
 
-    addProperty "elastic.username" "${ELASTIC_SEARCH_USERNAME}" ${JFMC_PROPERTIES} && \
-    addProperty "elastic.password" "${ELASTIC_SEARCH_PASSWORD}" ${JFMC_PROPERTIES} && \
+    addProperty "elastic.username" "${JF_SHARED_ELASTICSEARCH_USERNAME}" ${JFMC_PROPERTIES} && \
```

Comment:
Is this still needed? I think JFMC can use JF_SHARED_ELASTICSEARCH_PASSWORD directly

Reply:
Yes, Not needed,  Removed  :)